lib/src/code_field/span_builder.dart

```diff
@@ -65,25 +66,25 @@ class SpanBuilder {
       children: _buildList(
         nodes: node.children,
         theme: theme,
-        parentStyle: style ?? parentStyle,
+        ancestorStyle: classStyle ?? ancestorStyle,
       ),
       style: paledStyle,
```

Comment:
`paledStyle` is not a good name because it can be not pale.

`style: _paleIfRequied(style),` ?

Reply:
It isn't good idea to pale style after updating line index. Renamed variable